server/config/config.js

```diff
@@ -0,0 +1,27 @@
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+const dialect = 'postgres';
+
+module.exports = {
+  development: {
+    username: process.env.DB_USERNAME,
+    password: process.env.DB_PASSWORD,
+    database: process.env.DEV_DATABASE,
+    host: process.env.DB_HOST,
+    port: process.env.DB_PORT,
+    dialect,
```

Comment:
You can as well add operatorsAliases: false, after the end of this line to solve sequelize depreciation issues.